src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -70,4 +70,4 @@ cell_diffusion_timescale(::ConvectiveAdjustmentVerticalDiffusivity{<:VerticallyI
                          diffusivities, grid) = Inf
 
 cell_diffusion_timescale(closure::Tuple, diffusivities, grid) =
-    min(Tuple(cell_diffusion_timescale(c, diffusivities, grid) for c in closure)...)
+    min(Tuple(cell_diffusion_timescale(c, diff, grid) for (c, diff) in zip(closure, diffusivities))...)
```

Comment:
```suggestion
    min(Tuple(cell_diffusion_timescale(c, diffusivity, grid) for (c, diffusivity) in zip(closure, diffusivities))...)
```

Reply:
I'm reluctant to change from `diff` to `diffusivity` because in some cases (for example AMD) `diff` is actually a NamedTuple that can contain several diffusivities. So I'm afraid the code will be a tad misleading in that sense.

Although I agree that naming it `diff` isn't great either...

Reply:
Something more verbose?
diffusivity_coefficient?

Reply:
You don't need `Tuple`

Reply:
Went with

```julia
cell_diffusion_timescale(closure::Tuple, diffusivity_fields, grid) =
    minimum(cell_diffusion_timescale(c, diffusivities, grid) for (c, diffusivities) in zip(closure, diffusivity_fields))
```

`diffusivity_fields` is the name of the tutple in `model`, so it works well, and `diffusivities` is the name of that variable in the function `cell_diffusion_timescale()`, so it also works out well. I think this is clear